xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-solr/xwiki-platform-search-solr-server/xwiki-platform-search-solr-server-data/src/main/resources/xwiki/conf/schema.xml

```diff
@@ -242,7 +242,7 @@
    <dynamicField name="*_ar"  type="text_ar"    indexed="true"  stored="true" multiValued="true" />
    <dynamicField name="*_bg"  type="text_bg"    indexed="true"  stored="true" multiValued="true" />
    <dynamicField name="*_ca"  type="text_ca"    indexed="true"  stored="true" multiValued="true" />
-   <dynamicField name="*_cz"  type="text_cz"    indexed="true"  stored="true" multiValued="true" />
+   <dynamicField name="*_cs"  type="text_cz"    indexed="true"  stored="true" multiValued="true" />
```

Comment:
Maybe it would also be useful to rename the type "text_cz" into "text_cs", both in the usage (here) and in its definition, for correctness.